Docs/credits/Index.md

```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax), [Discord](https://discordapp.com/users/730358289849778186)
+
+### Maintiners
```

Comment:
Please alphabetise this section and remove the Discord accounts for privacy reasons.

Reply:
Alright